Dockerfile.util

```diff
@@ -0,0 +1,26 @@
+FROM senzing/senzingsdk-runtime:4.0.0
```

Comment:
I don't think we need another "utility" container. Is the tools container missing something that you need?